deploy/helm/kuberhealthy/values.yaml

```
@@ -103,7 +103,7 @@ check:
       MAX_FAILURES_ALLOWED: "10"
     nodeSelector: {}
   podStatus:
-    enabled: false
+    enabled: true
```

```suggestion
    enabled: false
```

I agree with @jonnydawg - let's be really careful about what default functionality we ship.  We should also be careful when adding additional checks by default because users updating may be surprised by the additional checks.

---

Thanks, will be more careful next time.